i18n/es-ES.json

```
@@ -1 +1,104 @@
-{}
\ No newline at end of file
+{
+  "POWERCORD_ACCOUNT": "Cuenta de Powercord",
+  "POWERCORD_GENERAL_SETTINGS": "Configuración general",
+  "POWERCORD_REFRESH_ACCOUNTS": "Actualizar cuentas",
+  "POWERCORD_PASSPHRASE": "Frase de contraseña",
+  "POWERCORD_UPDATE_PASSPHRASE": "Actualizar frase de contraseña",
+  "POWERCORD_UPDATE_PASSPHRASE_MODAL1": "Esta frase de contraseña se usará para cifrar sus datos antes de enviarlos a los servidores de Powercord. Se recomienda configurar uno, pero puede dejarlo vacío para enviar sus datos sin cifrar.",
+  "POWERCORD_UPDATE_PASSPHRASE_MODAL2": "Si ya está utilizando la sincronización en otras máquinas, coloque la misma frase de contraseña que usó. **El uso de otra frase de contraseña sobrescribirá los datos antiguos, así que tenga cuidado**.",
+  "POWERCORD_UNLINK": "Desconectar",
+  "POWERCORD_LINK_NOW": "Vinculalo ahora",
+  "POWERCORD_LINKING_UNLINKED": "Aún no ha vinculado su cuenta.",
+  "POWERCORD_LINKING_WAITING": "Vinculación de su cuenta...",
+  "POWERCORD_LINKING_TIMED_OUT": "Se agotó el tiempo de espera del enlace :( ¿Quizás intente de nuevo?",
+  "POWERCORD_LINKING_ERRORED": "Se produjo un error. Verifique la consola para obtener más detalles y, por favor, [repórtelo] ({newIssueUrl}) si vuelve a ocurrir.",
```

Space should be removed
```suggestion
  "POWERCORD_LINKING_ERRORED": "Se produjo un error. Verifique la consola para obtener más detalles y, por favor, [repórtelo]({newIssueUrl}) si vuelve a ocurrir.",
```